1.Core/Helpers/ProjectContextMenuHelper.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Linq;
+
+namespace Todoly.Core.Helpers;
+
+public class ProjectContextMenuHelper
+{
+    public static string ParseButtonName(string name)
+    {
+        Dictionary<string, string> contextMenuOptions = new Dictionary<string, string>() {
+            { "Edit", "edit" },
+            { "Add item above", "add separator" },
+            { "Add item below", "add" },
+            { "Delete", "delete" }
+        };
+
+        string res = contextMenuOptions.GetValueOrDefault(name)!;
+
+        if (res != null)
+        {
+            return res;
+        }
+        else
+        {
+            throw new ArgumentException("Invalid context menu option");
```

Comment:
This exception can be more informative if you include the argument passed to the method: `Invalid context menu option, received: {name}`

Reply:
You're right.